tensorflow/java/src/main/java/org/tensorflow/VariableInput.java

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Just to clarify quickly, VariableInputs are to be used/instantiated by any operations dealing with variable references, not only by Variable. But I can plan a separate PR for everything related to variables if you prefer, that's ok with me too.